assets/js/pages/ClusterDetails/ClusterDetails.jsx

```diff
@@ -119,6 +119,21 @@ function ClusterDetails({
             }}
             onCancel={closeOperationModal}
           />
+          <SimpleAcceptanceOperationModal
```

Comment:
Just moving this inside `{operationsEnabled && (` condition, so we can easier disable if needed. (some buttons are still not guarded by this variable)